x/relationships/client/cli/query.go

```diff
@@ -72,3 +73,25 @@ func GetCmdQueryUserRelationships(cdc *codec.Codec) *cobra.Command {
 		},
 	}
 }
+
+func GetCmdQueryUserBlocks(cdc *codec.Codec) *cobra.Command {
+	return &cobra.Command{
+		Use:   "user-blocks [address]",
```

Comment:
I would suggest renaming this to `blacklist` since it's a common term referring to the list of all blocked users.

```suggestion
		Use:   "blacklist [address]",
```